tests/cli-v2.rs

```diff
@@ -939,7 +937,7 @@ fn update_unavailable_std() {
             config,
             &["rustup", "update", "nightly", "--no-self-update"],
             &format!(
-                "component 'rust-std' for target '{}' is unavailable for download for channel 'nightly'",
+                "component 'rust-std' for target '{}' is unavailable for download",
```

Comment:
This loss of information means we've broken one of the error messages and that needs attention